tough-kms/CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.11.0] - 2024-10-10
+### Changes
+- Fix 128bit integer serialization (thanks @aumetra) [#820]
```

Comment:
I don't think this is relevant to `tough-kms`'s changelog.

Reply:
Done.